src/app/components/ms-desc-selector/ms-desc-selector.component.html

```diff
@@ -7,5 +7,5 @@
     [items]="msDesc$ | async" #ngSelectComponent
     (clear)="resetMsDesc();"
     [placeholder]="'selectMsDesc' | translate" 
-    (change)="msDescID = $event.id; openMsDescContent()">     
+    (change)="msDescID = $event?.id; openMsDescContent()">     
```

Comment:
Is it possible to get an undefined event on change event occurrence?

Reply:
Yes, you can unselect everything (in this case the box containing msDesc will be close). The change event of ng-select will output the whole model, thus if you unselect everything, the model will be undefined.